client/src/components/CampaignsMap.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import Modal from 'react-modal';
+import { withGoogleMap, GoogleMap, Marker } from "react-google-maps";
+import withScriptjs from "react-google-maps/lib/async/withScriptjs";
+import * as _ from "lodash";
+
+// Wrap all `react-google-maps` components with `withGoogleMap` HOC
+// and name it GettingStartedGoogleMap
+const GettingStartedGoogleMap = withScriptjs(
+  withGoogleMap(props => (
+    <GoogleMap
+      ref={props.onMapLoad}
+      defaultZoom={12}
+      defaultCenter={{ lat: 39.7392, lng: -104.9903 }}
+      onClick={props.onMapClick}
+    >
+      {props.markers.map(marker => (
+        <Marker
+          {...marker}
+          onRightClick={() => props.onMarkerRightClick(marker)}
+        />
+      ))}
+    </GoogleMap>
+)));
+class ApartmentMap extends Component {
```

Comment:
Maybe make this a functional component to keep it simple? https://stackoverflow.com/a/36137801